.env.example

```
@@ -1,4 +1,13 @@
 BACKEND_URL=http://localhost:8000
 GUARDRAILS_URL = http://localhost:8001
-GUARDRAILS_TOKEN = 
+GUARDRAILS_TOKEN =
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)